A lot of new endpoints

gcouti opened the pull request:
Hello, 

I create a lot of new endpoints and split it in other files. Please accept this pull request. 

Thank you

Commit:
Saving state on current changes.

gcouti committed:
New functions for greenhopper endpoints.

Two functions:

- List sprints
- Get informations about on sprint

Module for issues

Clean up code

- Separate modules

Test files

New fields in issues type

Debug, variables

New JIRA fields

Not used import

New features

Time correct format

Time

Update worklog for issues

Fix test and running go fmt

Remove comments

gcouti committed:
New functions for greenhopper endpoints.

Two functions:

- List sprints
- Get informations about on sprint

gcouti committed:
New functions for greenhopper endpoints.

Two functions:

- List sprints
- Get informations about on sprint

gcouti committed:
Module for issues

gcouti committed:
Clean up code

- Separate modules

gcouti committed:
Test files

gcouti committed:
New fields in issues type

gcouti committed:
Debug, variables

gcouti committed:
New JIRA fields

gcouti committed:
Not used import

gcouti committed:
New features

gcouti committed:
Time correct format

gcouti committed:
Time

gcouti committed:
Update worklog for issues

gcouti committed:
Merge branch 'master' of https://github.com/plouc/go-jira-client

rande commented on issues.go:
```diff
@@ -0,0 +1,424 @@
+		fmt.Println("%s", err)
+	}
+	//
+	//    for _, issue := range issues.Issues {
```
Why this code is commented ?

gcouti replied:
This is not necessary, because umarshal already do that.

rande commented on issues_test.go:
```diff
@@ -0,0 +1,83 @@
+    "fmt"
+    "testing"
+    "encoding/json"
+    "github.com/hekima/go-jira-client"
```
This is wrong, tests are failing due to this import

rande commented:
Can you squash you commits ? also can you run `go fmt` or make format to respect CS ? Also can you make sure there is no useless code as comment. Thanks

gcouti committed:
Fix test and running go fmt

gcouti committed:
Remove comments

gcouti committed:
New functions for greenhopper endpoints.

Two functions:

- List sprints
- Get informations about on sprint

Module for issues

Clean up code

- Separate modules

Test files

New fields in issues type

Debug, variables

New JIRA fields

Not used import

New features

Time correct format

Time

Update worklog for issues

Fix test and running go fmt

Remove comments

fmt and correcting merges
